src/component-library/CoinIcon/CoinIcon.tsx

```diff
@@ -1,3 +1,5 @@
+import { CurrencyIdLiteral } from '@interlay/interbtc-api';
```

Comment:
hi @crypto-engineer this doesn't work - I looked into this when I did the initial work, and Storybook won't run with imports from `interbtc-api`. I spent some time on it, but decided in the end not to have any `interbtc-api` dependencies in the component library.  Changes to the app are fine, but could you back out the changes to the component library?

Reply:
Might be something we can revisit when the currency types are simplified in the lib, but for now I'd prefer to keep it as it was (i.e. with the `Tokens` string literal type: `type Tokens = 'BTC' | 'DOT' | 'IBTC' | 'INTR' | 'KBTC' | 'KINT' | 'KSM';`)

Reply:
Done!